engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -3902,6 +3919,7 @@ private Option add(Option opt)
     final Option originalAppID = add(OptionBuilder.withArgName("application id").hasArg().withDescription("Specify original application identifier for restart.").create("originalAppId"));
     final Option exactMatch = add(new Option("exactMatch", "Only consider applications with exact app name"));
     final Option queue = add(OptionBuilder.withArgName("queue name").hasArg().withDescription("Specify the queue to launch the application").create("queue"));
+    final Option tags = add(OptionBuilder.withArgName("comma separated tags").hasArg().withDescription("Specify the tags for the application").create("tags"));
```

Comment:
Is there a more standards compliant way (posix) of specifying multiple value arguments so that the parser will itself handle it and return multiple values and we don't have to handle it specially on our side?